src/python/turicreate/test/test_image_similarity.py

```diff
@@ -63,19 +67,27 @@ def get_test_data():
         )
         images.append(tc_image)
 
-    return tc.SFrame({"awesome_image": images})
+    data_dict = {"awesome_image": images}
+    data = tc.SFrame(data_dict)
+
+    for model_name, feature_length in MODEL_TO_FEATURE_SIZE_MAPPING.items():
+        if str(feature_length) in data_dict.keys():
+            continue
+        data[str(feature_length)] = get_deep_features(data["awesome_image"], model_name)
+
+    return data
```

Comment:
Same comment here as I made in `test_image_classifier.py`

Reply:
Same comment as above.